catalog/spatial/registry/registry-schema-bindings/src/main/java/org/codice/ddf/registry/schemabindings/helper/RegistryNamespacePrefixMapper.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+
+package org.codice.ddf.registry.schemabindings.helper;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import javax.xml.XMLConstants;
+
+import org.codice.ddf.spatial.ogc.csw.catalog.common.CswConstants;
+
+import com.sun.xml.bind.marshaller.NamespacePrefixMapper;
+
+public class RegistryNamespacePrefixMapper extends NamespacePrefixMapper {
+
+    public static final String XS_NAMESPACE = "xs";
+
+    public static final String DATE_TIME = XS_NAMESPACE.concat(":dateTime");
+
+    private static final String WRS_NAMESPACE = "wrs";
+
+    private Map<String, String> namespaceMap = new HashMap<>();
+
+    public RegistryNamespacePrefixMapper() {
+        namespaceMap.put(XMLConstants.W3C_XML_SCHEMA_INSTANCE_NS_URI, XS_NAMESPACE);
+        namespaceMap.put("http://www.opengis.net/cat/wrs/1.0", WRS_NAMESPACE);
+        namespaceMap.put(CswConstants.EBRIM_SCHEMA, CswConstants.EBRIM_NAMESPACE_PREFIX);
+        namespaceMap.put(CswConstants.OGC_SCHEMA, CswConstants.OGC_NAMESPACE_PREFIX);
+        namespaceMap.put(CswConstants.GML_SCHEMA, CswConstants.GML_NAMESPACE_PREFIX);
+    }
+
+    @Override
+    public String getPreferredPrefix(String s, String s1, boolean b) {
+        return namespaceMap.getOrDefault(s, s1);
```

Comment:
Does that mean:
if (b)
  namespaceMap.get(s)
else
  namespaceMap.getOrDefault(s, s1)
?